### DISCUSSION TITLE
About presets configs

### DISCUSSION THREAD

**Discussion by Zanetti96** [darthcloud/BlueRetro#678] [Q&A] [answered]
Hello! I have one blueretro for my N64 and i was testing the 007 goldeneye preset, which works very well, but when i load this preset, all games get the same configuration, so i need to reset the device to play another game with normal buttons. Is there a way to put this preset only for a specific game? Or maybe a way to fast change the presets, like a combo input

**Comment by darthcloud** [comment]
If you got an Everdrive, the gameid it send can be used to save a specific config/preset for that game only.

See docs: https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#21---config-selection

**Reply by Zanetti96** [reply]
I use a everdrive but when i enter on the preset settings, it won't recognize the gameid, saying: undefined (n64)

**Reply by darthcloud** [reply]
Did you launch the game first?

**Reply by Zanetti96** [reply]
Yes, what I do:

Turn on the console with blueretro in
Turn on the controller (ds4)
Open the game through the everdrive menu
Turn off the controller by holding the ps button
Open blueretro.io, connect on my cellphone and go to preset settings

**Reply by darthcloud** [reply]
Are you sure the everdrive FW version you got support gameid?

**Reply by Zanetti96** [reply]
It's a ed64 plus, but idk if it supports gameid, don't find it anywhere

**Reply by darthcloud** [reply, marked as answer]
ok that ED64 look completely unrelated to krikzz's everdrive.

Its just using a similar name to confuse people.

So its unlikely to support gameid.

**Reply by Zanetti96** [reply]
Alright, thanks for the help!!
